docs/connections.md

```diff
@@ -242,7 +242,7 @@ data-validation connections add
     --database DATABASE                                 Redshift database
 ```
 
-## FileSystem (CSV or JSON only)
+## FileSystem (CSV, ORC, PARQUET or JSON only)
```

Comment:
Besides the section title, line 253 also needs to be updated to add the new file types